lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -17,10 +17,14 @@ class GithubRelease # rubocop:disable Metrics/ClassLength
     def_delegator :@build_mechanism, :output_file
 
     # @param build_mechanism Delegates building after GitHub release is created.
-    def initialize(build_mechanism, ci_status_timeout: 600, max_tag_find_timeout: 240)
+    def initialize(build_mechanism,
+                   ci_status_timeout: 600,
+                   max_tag_find_timeout: 240,
+                   skip_ci_status: false)
       @build_mechanism = build_mechanism
       @ci_status_timeout = ci_status_timeout
       @max_tag_find_timeout = max_tag_find_timeout
+      @skip_ci_status = skip_ci_status.nil? ? Moonshot.config.skip_ci_status : skip_ci_status
```

Comment:
We default to false, which means that by default we won't use the Moonshot.config.skip_ci_status unless someone passes nil as a keyword argument here. If we implement CLI hooks on the build mechanisms, we can modify the instance state using the command line argument lambda of OptionParser, and allow a keyword argument to override the default.

Reply:
Default argument value should be nil..  now Fixed